servicetalk-traffic-resilience-http/src/main/java/io/servicetalk/traffic/resilience/http/AbstractTrafficResilienceHttpFilter.java

```diff
@@ -280,6 +287,27 @@ public void cancel() {
                         }
                     }
                 }, true));
+        if (ENABLE_DEBUGGING) {
+            response = response.liftSync(subscriber -> new SingleSource.Subscriber<StreamingHttpResponse>() {
+                @Override
+                public void onSubscribe(Cancellable cancellable) {
+                    subscriber.onSubscribe(cancellable);
+                }
+
+                @Override
+                public void onSuccess(@Nullable StreamingHttpResponse result) {
+                    subscriber.onSuccess(result);
+                    DebugTrackingDelegatingTicket.setTerminationCause(ticket, result == null ? null : result.status());
```

Comment:
Should we hold the whole the whole `response`? That way we can see more if it's on the heap dump or log more inside `finalize()` method

Reply:
So long as our `response.toString()` method won't leak anything sensitive into the logs that seems fine to me.

Reply:
It logs only `version() + " " + status()`